q2_composition/plugin_setup.py

```diff
@@ -198,7 +199,8 @@
                         "this index."),
         'level_delimiter': ("If feature ids encode hierarchical information, "
                             "split the levels when generating feature labels "
-                            "in the visualization using this delimiter.")},
+                            "in the visualization using this delimiter."),
+        'label_limit': ("Set labelLimit for y-axis labels")},
```

Comment:
Remember that this documentation is intended for end users, not developers. In most cases end users won't know what `labelLimit` is, so your documentation should describe the effect that setting this parameter will have rather than what is technically happening behind the scenes. 

```suggestion
        'label_limit': ("Set the maximum length that will be viewable for axis labels. "
                        "You can set this parameter if your axis labels are being cut off.")},
```